fpna-template/Constants/FpnaCalculationEngine.ipynb

```diff
@@ -0,0 +1,33 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "dbO68puCAEuKRWoipkS-2Q",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "code",
+            "source": [
+                "var projectName = \"fp-ce-oct23\";",
```

Comment:
I would leave fpna here as name of the project

Reply:
changed